src/components/common-components/LargeTextBlock/LargeTextBlock.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import './LargeTextBlock.css';
+import Fade from 'react-reveal/Fade';
+
+/**
+ * @returns {React.Component}
+ */
+const LargeTextBlock = props => {
+
+    const [expanded, toggleExpand] = useState(false);
+    const [hover, toggleHover] = useState(false);
+
+    return <>
+        <div 
+            className="large-text-div"
+            onClick={() => toggleExpand(!expanded)}
+            onMouseEnter={() => toggleHover(true)}
+            onMouseLeave={() => toggleHover(false)}
+            style={{
+                backgroundColor: props.clickable && (hover || expanded) ? '#4D2393' : '#f0eafa',
+                transition: 'background-color 0.5s, color 0.5s',
```

Comment:
maybe you want variables for these lengths as well so you can change them later on easy